app/javascript/app/components/countries-menu/countries-menu-actions.js

```diff
@@ -0,0 +1,7 @@
+import { createAction } from 'redux-actions';
+
+const openCountriesMenu = createAction('openCountriesMenu');
```

Comment:
I think we will need to use component state for this for performance. With a simple state show:true/false to change a class which affects the opacity. I am finding it very slow.